### PR TITLE
Issue 1 2x ember

### PR DESCRIPTION
**Pull request by JennyJudova**
Issue 1 - 2x emberjs, ember data, ember cli entries.

### CODE REVIEW

**Comment by JennyJudova** [comment]
@kennethlarsen @abhilashlr this is review ready :)

File: source/ember-data-changes/3.12.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ changes:
     title: "Evented Api Usage"
     link: "https://blog.emberjs.com/2019/08/16/ember-3-12-released.html"
   -
-    eprecation: true
+    deprecation: true
```
Comment (abhilashlr) [inline source/ember-data-changes/3.12.md:9]: Thanks for fixing this @JennyJudova 👍

File: source/ember-js-changes/2.14.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    title: "Ember.MODEL_FACTORY_INJECTIONS"
+    link: "https://blog.emberjs.com/2017/07/06/ember-2-14-released.html"
+  -
+    deprecation: true
+    title: "eventManager property and canDispatchToEventManager property"
+    link: "https://blog.emberjs.com/2017/07/06/ember-2-14-released.html"
+---
+
+
+
```
Comment (abhilashlr) [inline source/ember-js-changes/2.14.md:15]: Can we remove these extra spaces please?

Comment (JennyJudova) [reply source/ember-js-changes/2.14.md]: done

File: source/ember-data-changes/3.12.md (hunk 1 above)
Comment (JennyJudova) [reply source/ember-data-changes/3.12.md:9]: :)

**Pull request merged**